File: BehsaApp/BehsaApp.Web/Modules/Behsa/Employees/EmployeesForm.cs
using Serenity.ComponentModel;
using System;

namespace BehsaApp.Behsa.Forms;

[FormScript("Behsa.Employees")]
[BasedOnRow(typeof(EmployeesRow), CheckNames = true)]
public class EmployeesForm
{
    public string NationalCode { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public DateTime BirthDate { get; set; }
}

File: BehsaApp/BehsaApp.Web/Modules/ServerTypes/Behsa/EmployeesForm.ts
import { StringEditor, DateEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface EmployeesForm {
    NationalCode: StringEditor;
    FirstName: StringEditor;
    LastName: StringEditor;
    BirthDate: DateEditor;
}

export class EmployeesForm extends PrefixedContext {
    static formKey = 'Behsa.Employees';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!EmployeesForm.init)  {
            EmployeesForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;

            initFormType(EmployeesForm, [
                'NationalCode', w0,
                'FirstName', w0,
                'LastName', w0,
                'BirthDate', w1
            ]);
        }
    }
}